### PR TITLE
SciGlassCalorimeter: specify tower material in the compact file

### PR DESCRIPTION
**Pull request by veprbl**
### Briefly, what does this PR introduce?
Introduces an option to specify the tower material.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No

### Does this PR change default behavior?
No

### CODE REVIEW

File: compact/ecal/barrel_sciglass.xml
Comment (wdconinc) [inline compact/ecal/barrel_sciglass.xml:50]: A more typical approach is to add `material` as an attribute to the structure that defines the object that should receive that material. Without that, this sort of stands on its own without context so it may be unclear what the material exactly refers to.

Comment (veprbl) [reply compact/ecal/barrel_sciglass.xml:50]: Yeah, thought about this. The other option is to put the material as an attribute into each family, or add a `<towers />` tag.

**Review by wdconinc — APPROVED** [review]
I'll approve this and leave it up to you as to whether you want to define materials for individual parts of the geometry.

**Comment by veprbl** [comment]
I don't think we'd want or need to define materials for each tower family.

Another question would be to see how the material definitions could be kept more uniform across all the other calorimeters. I'd move forward with this option, as it's quite niche one, we can always change things later.

**Comment by wdconinc** [comment]
I'm all for uniformity in material definitions! 

I'll just force-merge this rather than wait for another pipeline sequence after #188 was merged.

**Pull request merged**
